common/gossmods_listpeerchannels.c

```diff
@@ -155,7 +162,7 @@ gossmods_from_listpeerchannels_(const tal_t *ctx,
 		/* We add both directions */
 		cb(mods, self, &dst, &scidd, capacity_msat,
 		   htlc_min[LOCAL], htlc_max[LOCAL],
-		   spendable, fee_base[LOCAL], fee_proportional[LOCAL],
+		   spendable, max_total_out_htlc, fee_base[LOCAL], fee_proportional[LOCAL],
```

Comment:
just to confirm my understanding: `max_total_out_htlc` is the total amount that we can receive, so it gets added as the total max on the local receiving channel; `max_total_in_htlc` is the total amount we can send, so it gets added to the `REMOTE` set.

maybe renaming these would make this clearer? `max_total_out` seems to imply amount we can receive and vice versa for `total_in` lol

Reply:
@rustyrussell: help!

From the spec BOLT02:
```
A sending node:
    - if the total value of offered HTLCs would exceed the remote's `max_htlc_value_in_flight_msat`:
        - MUST NOT add an HTLC.

A receiving node:
    - if a sending node adds more than receiver `max_value_in_flight_msat` worth of offered HTLCs to its local commitment transaction:
        - SHOULD send a `warning` and close the connection, or send an `error` and fail the channel.
```

My interpretation is that `our_max_value_in_flight_msat` limits how much can we receive, and `their_max_value_in_flight_max` limits how much can we send.